autoload/vebugger.vim

```diff
@@ -268,26 +270,39 @@ function! vebugger#createDebugger(command)
     return l:debugger
 endfunction
 
+if has('timers')
+    function! ReadingTimerCallback(debugger, timer_id)
```

Comment:
Yea, Vimscript is weird like that... I would actually expect `s:foo` will be the function pointer - or at the very least `s:.foo`...